support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -78,6 +84,18 @@ class CustomActionBuilders(
   def maybeAuthenticatedAction(identityClientId: String = membersIdentityClientId): ActionBuilder[OptionalAuthRequest, AnyContent] =
     PrivateAction andThen maybeAuthenticated(onUnauthenticated(identityClientId))
 
+  def alarmOnFailure[A](action: Action[A]): EssentialAction =
+    (v1: RequestHeader) => action.apply(v1).map { result =>
+      if (result.header.status.toString.head != '2') {
+        SafeLogger.error(scrub"pushing alarm metric - non 2xx response ${result.toString()}")
```

Comment:
the error log contains something like this
```
2021-09-21T16:38:40,122+0000 [application-akka.actor.default-dispatcher-17] INFO  com.gu.monitoring.SafeLogger$ - incoming POST: {"firstName:"kaejgrkwe","lastName":"kaejgrkwe","email":"kaejgrkwe@gu.com","billingAddress":{"lineOne":null,"lineTwo":null,"city":null,"state":null,"postCode":null,"country":"GB"},"deliveryAddress":null,"product":{"productType":"Contribution","amount":20,"currency":"GBP","billingPeriod":"Monthly"},"firstDeliveryDate":null,"paymentFields":{"paymentMethod":"pm_0JcBsZItVxyc3Q6nCLSfvaXk","stripePaymentType":"StripeCheckout"},"ophanIds":{"pageviewId":"ktuao3s12md8dttf58ww","browserId":null,"visitId":null},"referrerAcquisitionData":{"queryParameters":[]},"supportAbTests":[{"name":"productSetTest","variant":"control"},{"name":"GBP_COUNTRIES_AMOUNTS_TEST","variant":"CONTROL"}],"telephoneNumber":null,"debugInfo":"contributions does not collect redux state"}
2021-09-21T16:38:40,122+0000 [application-akka.actor.default-dispatcher-18] ERROR com.gu.monitoring.SafeLogger$ - pushing alarm metric - non 2xx response Result(400, TreeMap())

```
and the client side gets something like this
```
< HTTP/1.1 400 Bad Request
[....]
ParsingFailure: expected : got 'kaejgr...' (line 1, column 14)
```